app.rambox.ramboxce.yaml

```
@@ -0,0 +1,57 @@
+app-id: app.rambox.ramboxce
```

Perhaps better (would require renaming this file accordingly):
```suggestion
app-id: app.rambox.Rambox
```

---

In the free version, the package name is `Rambox`, the compiled binary is `rambox`, the rpm/deb installation directory is `/opt/Rambox`, the menu entry is `rambox.desktop`, the icons are `rambox.png`, etc.

In the pro version, the package name is `RamboxPro`, the compiled binary is `ramboxpro`, the rpm/deb installation directory is `/opt/RamboxPro`, the menu entry is `ramboxpro.desktop`, the icons are `ramboxpro.png`, etc.

Therefore, `app.rambox.Rambox` makes perfect sense here.